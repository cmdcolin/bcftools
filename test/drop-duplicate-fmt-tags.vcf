##fileformat=VCFv4.1
##FORMAT=<ID=GT,Number=1,Type=String>
##FORMAT=<ID=X,Number=1,Type=Integer>
##contig=<ID=chr1>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	A
chr1	1	.	A	C	.	.	.	GT:X:GT:X	0/0:0:0/1:1
