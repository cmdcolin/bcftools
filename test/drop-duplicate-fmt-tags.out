chr1	1	.	A	C	.	.	.	GT:X	0/0:0
